### PR TITLE
[iOS] Not log callback cache info if cache is empty

### PR DESCRIPTION
**Pull request by zhongwuzw**
If we don't use callback cache, every time we start the app, the log prints "Runner[667:97662] [VERBOSE0:callback_cache.cc(136)] Could not parse callback cache, aborting restore", which may lead to misleading.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt. See [testing the engine] for instructions on
writing and running engine tests.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I signed the [CLA].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[C++, Objective-C, Java style guides]: https://github.com/flutter/engine/blob/main/CONTRIBUTING.md#style
[testing the engine]: https://github.com/flutter/flutter/wiki/Testing-the-engine
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Review by bkonyi — APPROVED** [review]

**Comment by bkonyi** [comment]
Thanks for the contribution! Just waiting for sign-off from @Hixie to see if we're okay not adding a test for this.

**Comment by Hixie** [comment]
Ideally we would repurpose some of our integration tests over on the framework repo to trigger the condition that results in this output, and verify that the logs are entirely clear of extraneous messages. We have a few tests that check things like this already, at least on Android. cc @jmagman who may know precisely which tests would be best suited to use for this.

**Comment by jmagman** [comment]
I'm not familiar with how this cache works to know which tests might catch this, what are the cases where would it be empty?

**Comment by zhongwuzw** [comment]
> I'm not familiar with how this cache works to know which tests might catch this, what are the cases where would it be empty?

Seems most cases of users should be empty, only when navigator restorable method use it , like https://github.com/flutter/flutter/blob/d9a2229babbc7e28cabd0ad42470c527a6cd9d3e/packages/flutter/lib/src/widgets/navigator.dart#L5386、https://github.com/flutter/flutter/blob/d9a2229babbc7e28cabd0ad42470c527a6cd9d3e/packages/flutter/lib/src/widgets/navigator.dart#L4461 whitch calles `ui.PluginUtilities.getCallbackHandle`

**Review by chinmaygarde — APPROVED** [review]
Even though this is a verbose log, I don't think it adds any value. I'd be down to just removing it as it isn't actionable.

**Comment by jmagman** [comment]
> Even though this is a verbose log, I don't think it adds any value. I'd be down to just removing it as it isn't actionable.

SGTM, @zhongwuzw can you remove the `Could not parse callback cache` log?

**Comment by zhongwuzw** [comment]
> > Even though this is a verbose log, I don't think it adds any value. I'd be down to just removing it as it isn't actionable.
> 
> SGTM, @zhongwuzw can you remove the `Could not parse callback cache` log?

@jmagman Done. :)

File: lib/ui/plugins/callback_cache.cc
Comment (jmagman) [inline lib/ui/plugins/callback_cache.cc:133]: Sorry to nit on such a small change, but the log was documenting what was happening here.  Can you change this to a comment?
```c++
    // Could not parse callback cache, aborting restore.
    return;
```

**Pull request merged**
